scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,191 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
+    def test_logpmf(self):
+        # Ground truth value from R dmvhyper
+        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
+        assert_allclose(vals1, -1.119814, rtol=1e-6)
+
+        vals2 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=0)
+        assert_allclose(vals2, np.NAN, rtol=1e-8)
+
+        vals3 = multivariate_hypergeom.logpmf(x=[-3, 4], m=[5, 10], n=7)
```

Comment:
```suggestion
        # x < 0
        vals3 = multivariate_hypergeom.logpmf(x=[-3, 4], m=[5, 10], n=7)
```

Reply:
val7 has x > m, but it also has m.sum != n. Would be good to have an independent check of x > m.

Reply:
You don't necessarily need these comments, but they help check that we're covering all the different cases.

Reply:
Comments like this would help check that all the bases are covered.

Reply:
Added!